### PR TITLE
Include new wavefront sensor

### PR DESCRIPTION
**Pull request by PabloRdrRbl**
Include new wavefront sensor model based TBD et al. 2021.

This will help to do that.

- [x] Add DM
- [x] Add WFS
- [ ] Add SW for WFS

### CODE REVIEW

**Comment by PabloRdrRbl** [comment]
I think the DM will got in a M4.

**Comment by PabloRdrRbl** [comment]
Once we are done with HW, we could implement the SW in this way...

**Comment by ivalaginja** [comment]
I think this is not quite there yet, I think we also need an M5.

**Comment by ivalaginja** [comment]
Nice! Now we just have to make sure this doesn't break anything, and then we can merge it.

File: coronograph.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 - entrance pupil
+- WFS
```
Comment (PabloRdrRbl) [inline coronograph.txt:2]: Can specify the type of WFS?

Comment (ivalaginja) [inline coronograph.txt:2]: Actually, the WFS should maybe be not right where the coronagraph is.

Comment (PabloRdrRbl) [reply coronograph.txt:2]: I see what you mean.

**Comment by davidbourgeois** [comment]
We can add apodizer on coronograph

**Comment by ivalaginja** [comment]
@davidbourgeois not sure yet whether it will be an APLC or some other coronagraph. But that's why we have the PR interface, @PabloRdrRbl can tell us what his coronagraph plans are ;)

**Pull request merged**